---
title: "Single-cell Amplified Genomes"
layout: single
author_profile: true
author: Ashley Bell
permalink: /research/ashley/sags
toc: true
toc_label: "Contents"
toc_icon: "cog"
---
## Abstract
90 % of the ocean’s biomass comprise of marine microorganisms [1](/research/ashley/sags/#references) where the SAR11 bacteria is the most ubiquitous, comprising of 30 % of all surface water cells and one of the most widespread organisms on Earth [2]. Its ability to harvest labile dissolved organic matter [3] and produce volatile organic compounds [4,5] plays an important role in global biogeochemistry. Its small genome size and complexity provides insight into the microbiological evolution and function of streamlined cells [6]. However, complete SAR11 whole genomes sequences are relatively rare due to their difficulty in culturing [7–9], leading to gaps in knowledge of SAR11 diversity. Here we show the advantage of single-cell amplified genomes in genomic analysis of the SAR11 clade. 451 SAR11 bacteria were isolated and sequenced using SAG technology. We found evidence suggesting the presence of two new phylogenetic clades of SAR11, expanding the numbers of niche habitats SAR11 inhabits [10]. Additionally, genomic analysis confirms the existence of multiple hypervariable regions [11,12] containing niche-defining genes [13]. Analysis of these HVRs revealed genes related to nutrient uptake and phage defense, consistent with previous studies advocating for a King of the Mountain ecological strategy [14]. Lastly, identification of viral signatures within SAR11 SAGs lead to confirmation of existing viral clades within SAR11. Our results demonstrate the value of SAGs to allow for clade wide studies of the SAR11 pangenome. We anticipate the usage of SAGs to become a powerful alternative for the study of difficult to isolate and culture organisms. 

## Single-cell Amplified Genomes
Single-cell Amplified Genomes (SAGs) are genomes isolated from single cell sorting techniques and are [whole genome amplified](https://en.wikipedia.org/wiki/Multiple_displacement_amplification) and sequenced (Eberwine et al. 2014). Organisms usually have only 1 or a few copies of their genomic DNA (Alberts et al. 2014), which is not enough to sequence. Amplification of this genomic DNA allows for pure samples of an organism to be sequenced without the need for culturing (Lasken 2007). This is particularly useful for organisms that are difficult to grow or are slow growing. 451 SAR11 SAGs are sequenced within this study. SAR11 is an aquatic bacteria and the most abundant microbe in the world’s oceans (S. J. Giovannoni 2017). Culturing this slow growing organism is difficult and SAGs provide an alternative to the genomic study of these ubiquitous organisms. 

## 451 SAR11 SAGs 
SAGs were assembled and phylogentics determined to allow for catagorisation of SAGs into [existing SAR11 phylogentic clades](https://www.annualreviews.org/doi/full/10.1146/annurev-marine-010814-015934). [Viral signatures](https://www.nature.com/articles/nature11921) within SAGs were extracted and analysed and [hypervariable regions](https://mbio.asm.org/content/3/5/e00252-12) determined.

## Phylogenetic Tree
Generally, phylogenetics measured by how related a species is to each other based on some kind of simularity, like their genetic content. Similarly grouped species sit within the same “branch” of a phylogenetic tree, with more distantly related species are then located further away. To establish relatedness, conserved genes of all species like the 16/18S rRNA is used. This is because all cells have a form of this 16/18S gene within their genome, and comparisons of similarity can then be performed between such individuals. Phylogenetics was used to first describe the relationship of each SAG to each other. This would allow for categorisation of each SAG into the established 5 clades of SAR11. Average Amino acid Identity [AAI](https://www.asmscience.org/content/journal/microbe/10.1128/microbe.9.111.1) was used to [confirm splits within the branch structure of phylogenetic trees](../../assets/images/SAR11_SAG_AAI.png). [Metagenomic data](https://www.nature.com/articles/sdata2018176) of differing locations within the ocean were then used to confirm splits within phylogentics trees corrispond with ecological niches. 

## Viral Signatures

## Hypervariable Regions



## References
1. 	Suttle CA. Marine viruses — major players in the global ecosystem [Internet]. Nature Reviews Microbiology. 2007. pp. 801–812. doi:10.1038/nrmicro1750
2. 	Morris RM, Rappé MS, Connon SA, Vergin KL, Siebold WA, Carlson CA, et al. SAR11 clade dominates ocean surface bacterioplankton communities. Nature. 2002;420: 806–810. doi:10.1038/nature01240
3. 	Carlson CA, Giovannoni SJ, Hansell DA, Goldberg SJ, Parsons R, Vergin K. Interactions among dissolved organic carbon, microbial processes, and community structure in the mesopelagic zone of the northwestern Sargasso Sea. Limnol Oceanogr. 2004;49: 1073–1083. doi:10.4319/lo.2004.49.4.1073
4. 	Sun J, Steindler L, Thrash JC, Halsey KH, Smith DP, Carter AE, et al. One carbon metabolism in SAR11 pelagic marine bacteria. PLoS One. 2011;6: e23973. doi:10.1371/journal.pone.0023973
5. 	Carini P, White AE, Campbell EO, Giovannoni SJ. Methane production by phosphate-starved SAR11 chemoheterotrophic marine bacteria. Nat Commun. 2014;5: 4346. doi:10.1038/ncomms5346
6. 	Giovannoni SJ, Cameron Thrash J, Temperton B. Implications of streamlining theory for microbial ecology. ISME J. 2014;8: 1553–1565. doi:10.1038/ismej.2014.60
7. 	Rappé MS, Connon SA, Vergin KL, Giovannoni SJ. Cultivation of the ubiquitous SAR11 marine bacterioplankton clade. Nature. 2002;418: 630–633. doi:10.1038/nature00917
8. 	Carini P, Campbell EO, Morré J, Sañudo-Wilhelmy SA, Thrash JC, Bennett SE, et al. Discovery of a SAR11 growth requirement for thiamin’s pyrimidine precursor and its distribution in the Sargasso Sea. ISME J. 2014;8: 1727–1738. doi:10.1038/ismej.2014.61
9. 	Carini P, Steindler L, Beszteri S, Giovannoni SJ. Nutrient requirements for growth of the extreme oligotroph “Candidatus Pelagibacter ubique” HTCC1062 on a defined medium [Internet]. The ISME Journal. 2013. pp. 592–602. doi:10.1038/ismej.2012.122
10. 	Giovannoni SJ. SAR11 Bacteria: The Most Abundant Plankton in the Oceans. Ann Rev Mar Sci. 2017;9: 231–255. doi:10.1146/annurev-marine-010814-015934
11. 	Wilhelm LJ, Tripp HJ, Givan SA, Smith DP, Giovannoni SJ. Natural variation in SAR11 marine bacterioplankton genomes inferred from metagenomic data. Biol Direct. 2007;2: 27. doi:10.1186/1745-6150-2-27
12. 	Grote J, Thrash JC, Huggett MJ, Landry ZC, Carini P, Giovannoni SJ, et al. Streamlining and core genome conservation among highly divergent members of the SAR11 clade. MBio. 2012;3. doi:10.1128/mBio.00252-12
13. 	Coleman ML, Sullivan MB, Martiny AC, Steglich C, Barry K, Delong EF, et al. Genomic islands and the ecology and evolution of Prochlorococcus. Science. 2006;311: 1768–1770. doi:10.1126/science.1122050
14. 	Giovannoni S, Temperton B, Zhao Y. Giovannoni et al. reply [Internet]. Nature. 2013. pp. E4–5. doi:10.1038/nature12388


